apache2/apache2_config.c

```diff
@@ -4045,12 +4060,21 @@ const command_rec module_directives[] = {
         CMD_SCOPE_ANY,
         "Set waf instanceId"
     ),
+#endif
+#ifndef _WIN32
+    AP_INIT_TAKE1 (
+        "SecWafLockUser",
+        cmd_waf_lock_user,
+        NULL,
+        CMD_SCOPE_ANY,
+        "Set waf lock user"
+    ),
     AP_INIT_TAKE1 (
-        "SecWafLockOwner",
-        cmd_waf_lock_owner,
+        "SecWafLockGroup",
+        cmd_waf_lock_group,
         NULL,
         CMD_SCOPE_ANY,
-        "Set waf lock owner"
+        "Set waf lock group"
     ),
 #endif
```

Comment:
Let's start marking `#endif` with the right end comments so that matching the `#ifndef` and `#endif` becomes a bit more readable.